pyvista/_vtk.py

```diff
@@ -0,0 +1,382 @@
+    from vtkmodules.vtkFiltersSources import (vtkOutlineCornerFilter,
+                                              vtkParametricFunctionSource,
+                                              vtkParametricFunctionSource,
+                                              vtkPlaneSource,
+                                              vtkArcSource,
+                                              vtkTessellatedBoxSource,
+                                              vtkOutlineCornerSource,
+                                              vtkCubeSource,
+                                              vtkArrowSource,
+                                              vtkCylinderSource,
+                                              vtkSphereSource,
+                                              vtkPlaneSource,
+                                              vtkLineSource,
+                                              vtkCubeSource,
+                                              vtkConeSource,
+                                              vtkDiskSource,
+                                              vtkRegularPolygonSource,
+                                              vtkLineSource,
+                                              vtkPointSource,
+                                              vtkArrowSource)
+    from vtkmodules.vtkFiltersGeometry import (vtkGeometryFilter,
+                                               vtkStructuredGridGeometryFilter,
+                                               vtkCompositeDataGeometryFilter,
+                                               vtkDataSetSurfaceFilter)
+    from vtkmodules.vtkFiltersExtraction import (vtkExtractEdges,
+                                                 vtkExtractGeometry,
+                                                 vtkExtractGrid,
+                                                 vtkExtractSelection)
+    from vtkmodules.vtkFiltersTexture import (vtkTextureMapToPlane,
+                                              vtkTextureMapToSphere)
+    from vtkmodules.vtkFiltersPoints import (vtkGaussianKernel,
+                                             vtkPointInterpolator)
+    from vtkmodules.vtkFiltersVerdict import (vtkCellSizeFilter,
+                                              vtkCellQuality)
+    from vtkmodules.vtkImagingGeneral import vtkImageGaussianSmooth
+    from vtkmodules.vtkImagingCore import (vtkExtractVOI,
+                                           vtkImageExtractComponents,
+                                           vtkImageDifference,
+                                           vtkRTAnalyticSource)
+    from vtkmodules.vtkFiltersFlowPaths import vtkStreamTracer
+    from vtkmodules.vtkCommonExecutionModel import vtkImageToStructuredGrid
+    from vtkmodules.numpy_interface.dataset_adapter import (VTKObjectWrapper,
+                                                            numpyTovtkDataArray,
+                                                            VTKArray)
```

Comment:
This appears to be identical to the import for VTK < 9 -- and same goes at least for the `numpy_support` imports. It would be nice to unindent the imports that are common to both code paths to make it clear what is version dependent and what's not

Reply:
The problem with using `vtk` instead of `vtkmodules` is importing anything from `vtk` loads the top level module, which in the case of `vtk`, means loading all the vtk libraries.  Do demonstrate, I first ran `time python -c "import pyvista"`:
```
time python -c "import pyvista"

real	0m0.289s
user	0m0.494s
sys	0m0.357s
```
Next, I placed the following at the top of the module:

```py
from vtk.numpy_interface.dataset_adapter import (VTKObjectWrapper,
                                                 VTKArray,
                                                 numpyTovtkDataArray)
```

which results in

```
time python -c "import pyvista"

real	0m0.334s
user	0m0.510s
sys	0m0.467s
```

If you go through `importime`, you'll also see all the modules being loaded.  Sadly, this won't work as a solution.

---

It's annoying for sure, but I think it's a far better solution rather than dropping vtk 8 support entirely, especially given the problems with closing render windows on vtk 9 on Linux.